lib/context.js

```diff
@@ -47,16 +49,19 @@ function SkaleContext(arg) {
   mkdirp.sync(this.basedir + 'stream');
 
   // Forward useful env in context for workers
-  for (var k = 0; k < envKeys.length; k++)
-    this.env[envKeys[k]] = process.env[envKeys[k]];
+  for(const k in envKeys) {
+    this.env[k] = process.env[k];
+  }
 
   this.once('connect', function(data) {
-    var i;
+
     process.title = 'skale-master_' + data.devices[0].wsid + ' ' + __filename;
-    if (!nworker || nworker > data.devices.length)
+    if (!nworker || nworker > data.devices.length) {
       nworker = data.devices.length;
+    }
+
     log('workers:', nworker);
-    for (i = 0; i < nworker; i++) {
+    for (var i = 0; i < nworker; i++) {
```

Comment:
use `let i = 0; ...` instead of `var i = 0; ...`, to set the scope of `i` to the loop bloc